bioio_ome_zarr/writers/ome_zarr_writer_v3.py

```diff
@@ -0,0 +1,497 @@
+import sys
+
+if sys.version_info < (3, 11):
+
+    class OMEZarrWriter:
+        """
+        Stub for Python <3.11. Instantiating this will always error.
+        """
+
+        def __init__(self, *args, **kwargs):
+            raise NotImplementedError(
+                "OMEZarrWriter requires Python 3.11+ and zarr>=3.0.6. "
+                "On Python 3.10 you can only use the basic writer."
```

Comment:
Whats the basic writer?

Reply:
it was the V2 writer. resolved in [2b11e82](https://github.com/bioio-devs/bioio-ome-zarr/pull/41/commits/2b11e82b50013e27943f10a1c66324b3bd096271)